### PR TITLE
WSL Pitfalls

### PR DESCRIPTION
**Pull request by awdeorio**
Centralized location for WSL pitfalls so that we can link to them.

### CODE REVIEW

File: docs/setup_wsl.md [hunk 1]
```diff
@@ -168,6 +168,52 @@ Enable "Use Ctrl+Shift+C/V Copy/Paste" option in the Console "Options" propertie
...
+### Root user
+Avoid doing everyday coding as the `root` user in WSL.  Some programs don't work correctly when run as `root`.  When you first installed Ubuntu, you should have been prompted to create a Linux username and password.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+**Bad example:** If the default is a root login, here's how to [change your linux username and password](https://docs.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password).  For the same reasons, avoid using `su` or `su root`.
+```console
+# whoami
```
Comment (melodell) [inline docs/setup_wsl.md:199]:
```suggestion
$ whoami
```

**Comment by noah-weingarden** [comment]
I suggest reverting commit 5b76c49: the `#` prefix is used in every terminal emulator I've seen when the logged-in user is root.

**Comment by awdeorio** [comment]
Bump @ohjuny !

**Comment by ohjuny** [comment]
LGTM, I think the $ prefix would be less confusing for students since it is consistent with all the other tutorials we have. @noah-weingarden thoughts?

**Comment by noah-weingarden** [comment]
I'm just not convinced that it's significant enough to confuse anyone meaningfully, so you might as well leave it as-is (the correct way).

**Comment by awdeorio** [comment]
I'm OK with this either way, so I'm going to merge.

**Pull request merged**